source/wasm/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "spirv-tools",
+  "version": "20.4.2",
```

Comment:
What does this version come from?
Normally the SPIRV-Tools version comes from the CHANGES file.

Reply:
Added logic (bash, sed) to take this from the CHANGES file and at the GITHUB_RUN_ID as patch part of.

So `v2020.5` from changes becomes e.g. `2020.5.2`

I also changed the publish step to use: https://github.com/marketplace/actions/npm-publish (for now with dry-run)